src/redist/redist.csproj

```diff
@@ -86,7 +86,7 @@
 
     <DotNetPublish ToolPath="$(PreviousStageDirectory)"
                    Configuration="$(Configuration)"
-                   ProjectPath="$(SrcDirectory)/tool_fsharp/tool_fsc.csproj" />
+                   ProjectPath="$(SrcDirectory)/tool_fsharp/tool_fsc.csproj" MSBuildArgs="/p:PublishDir=$(FSharpDirectory)" />
```

Comment:
Do the PublishFSharp changes affect the product?

Reply:
No, I made this change because on my local box, just setting DropSuffix was making FSharp still be published to the Suffix Based location. Moving the Publish parameter here caused it to be published to the right location.

I got nervous that we were going to have yet another failure when the tests ran.